repo/update-upstream.sh

```diff
@@ -22,8 +22,9 @@ cleanup() {
 ## Fetch the opam-repository
 
 cd "${WORK_DIR}"
-git clone --depth=1 ${OPAM_REPO} ${REPO_DIR_NAME}
+git clone ${OPAM_REPO} ${REPO_DIR_NAME}
 cd ${REPO_DIR_NAME}
+git checkout 1.2
```

Comment:
FWIW I think this change is equivalent to 
```
go clone --depth=1 --branch=1.2 ${OPAM_REPO} ${REPO_DIR_NAME}
```
which retains the shallow clone. Not sure if `1.2` here is a branch or a tag, but `--branch` is documented to accept either.

Reply:
Bah!

I suppose I could edit my previous comment to make you look like a loon but I won't ;-)